libs/ui-watt/src/lib/components/tooltip/watt-tooltip.component.scss

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use "sass:math";
+@use "@energinet-datahub/watt/utils" as watt;
+
+$arrow-size: 8px;
+$offset: 6px;
+
+:host {
+  z-index: 500;
```

Comment:
Is this high enough for use in modals? 😄

Reply:
It is, I've added an example of the tooltip to the modal showcase, so it's easy to test in the future.